### PR TITLE
Fix maxcompute ci job

### PR DESCRIPTION
**Pull request by typhoonzero**
The Maxcompute test cases are actually not running on travis ci.

`maxcompute.sh` is executed inside the Docker container, and we start the container by passing `-e SQLFLOW_TEST_DB_MAXCOMPUTE_AK=$MAXCOMPUTE_AK`, so use `SQLFLOW_TEST_DB_MAXCOMPUTE_AK` directly in the container.

### CODE REVIEW

**Comment by typhoonzero** [comment]
The current code of dealing table columns is fragile and hard to fix. Will try to polish it in this PR.

**Review by sneaxiy — COMMENTED** [review]
Please solve the conflict.

File: python/runtime/maxcompute.py [hunk 1]
```diff
@@ -32,8 +43,7 @@ def read_feature(raw_val, feature_spec):
                                         sep=feature_spec["delimiter"])
                 indices = indices.reshape(indices.size, 1)
                 values = np.ones([indices.size], dtype=np.int32)
-                dense_shape = np.array(feature_metas[name]["shape"],
-                                       dtype=np.int64)
+                dense_shape = np.array(feature_spec["shape"], dtype=np.int64)
```
Comment (sneaxiy) [inline python/runtime/maxcompute.py:46]: Seems that we can remove the `read_feature` method?

**Review by sneaxiy — APPROVED** [review]
LGTM with a tiny comment.

File: python/runtime/maxcompute.py (hunk 1 above)
Comment (typhoonzero) [reply python/runtime/maxcompute.py:46]: Will do in the next PR.

Comment (typhoonzero) [reply python/runtime/maxcompute.py:46]: `read_feature ` is called by `read_features_from_row` which will be used when predicting to get columns that are used as features to feed to the model.

**Pull request merged**